pkgs/development/compilers/crystal/default.nix

```diff
@@ -27,6 +27,8 @@
   tzdata,
   which,
   zlib,
+  # Configuration
```

Comment:
How about adding the upstream issue URL here? https://github.com/crystal-lang/crystal/issues/12299